src/adapter/http/tests/harness/client_side_harness.rs

```diff
@@ -101,17 +101,15 @@ impl ClientSideHarness {
     pub async fn pull_datasets(&self, dataset_ref: DatasetRefAny) -> Vec<PullResponse> {
         self.pull_service
             .pull_multi(
-                &mut vec![dataset_ref].into_iter(),
-                PullOptions {
+                vec![dataset_ref],
+                PullMultiOptions {
                     sync_options: SyncOptions {
                         create_if_not_exists: true,
                         ..SyncOptions::default()
                     },
-                    ..PullOptions::default()
+                    ..Default::default()
```

Comment:
Both are equivalent (accessing same method through type name vs through the trait) so I sometimes write `Type::default()` to be explicit, but the next day would write `Default::default()` to avoid name repetition. I'll revert back to type name, but yeah we need some convention :)

Reply:
The convention I would propose is:

Use explicit type name when passing default parameters to functions, e.g.:

```rust
pull_svc.pull(dataset, PullOptions::default())
```
This ensures that when method's second argument is refactored to a different type - client code will need to be updated and reconsidered (as opposed to passing `Default::default()` that might just work for new parameter type too.

Use `Default::default()` in struct initializers, e.g.:

```rust
pull_svc.pull(dataset, PullOptions {
  create_if_exists: true,
  ..Default::default()
})
```
This is to avoid type name repetition == less diff during renaming refactorings.